.dockerignore

```diff
@@ -57,11 +57,15 @@
 
 # Sphinx documentation
 docs/_build/
+.venv-docs/
 
 # PyBuilder
-target/
+**/target/
 
 # Virtual environment
-.env/
-.venv/
-venv/
+**/.env/
+**/.venv/
+**/venv/
+
+# local_dev mounted data
+local_dev/docker_data/
```

Comment:
`local_dev` implies to me local to the dev.  Perhaps anything inside of this directory should be ignored?

Reply:
sure 👍